lib/control.ml

```diff
@@ -44,12 +44,13 @@ let sequence {M : Monad} (ms : 'a M.t list) =
     ms
     (return [])
 
-(* Create a monad using default functor and idiom implementation *)
+(* Create a monad using default functor and applicative implementation *)
```

Comment:
Is this comment not still wrong? Shouldn't it be create Functor and Applicative implementation from Monad implementation?

Reply:
Ok no my bad, it is still creating a Monad, just letting you sidestep the the Functor and Applicative stages